lib/data/message.js

```diff
@@ -1,25 +1,23 @@
-"use strict";
+'use strict';
 
-function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
```

Comment:
This was very old Node syntax. By specifying our minimum Node version at 22, babel no longer needs to transpile into ES5 syntax.